modules/test/playwright/tests/object-web/main/objectFields.spec.ts

```diff
@@ -1089,4 +1089,55 @@ test.describe('Manage objectFields through Objects Admin UI', () => {
 			ERCValue
 		);
 	});
+
+	test('LPD-53450 - can delete created custom fields in a System Object', async ({
+		apiHelpers,
+		objectFieldsPage,
+		page,
+	}) => {
+		const objectDefinitionField =
+			await apiHelpers.buildRestClient(ObjectFieldAPI);
+
+		const fieldName = 'Custom Field';
+
+		const {items} =
+			await apiHelpers.objectAdmin.getSystemObjectDefinitions();
+
+		const systemObjectDefinition = items.find((item: ObjectDefinition) => {
```

Comment:
from what i understood here you are getting the first system object in the array and then adding a custom field to it right?

but i believe the poshi test does this same test for all the following objects:
- AccountEntry
- CPDefinition
- CommercePricingClass
- CommerceOrder
- Address
- User

did you align with the leadership to only do one? or is not necessary anymore? my suggestion is to talk with them to see if it's still necessary to test all system objects or only one is enough

Reply:
Hey @rebsilva, I talked to Gabriel and he agreed to simplify the test  to use just one system object.